### PR TITLE
Improve RestQL API

### PR DESCRIPTION
**Pull request by grainier-macrometa**
**Purpose**
- Fix restql drop query function (currently `null` is taken as the query name).
- Introduce userId as an UserQueryOptions field.

### CODE REVIEW

**Review by yehan-macrometa — APPROVED** [review]

**Review by StoyanTc — APPROVED** [review]
Please increment the latest value of the version by 1.

File: src/main/java/com/c8db/entity/UserQueryOptions.java [hunk 1]
```diff
@@ -26,10 +26,20 @@
  */
 public class UserQueryOptions {
 
+    private String userid;
     private String name;
     private Map<String, Object> parameter;
     private String value;
 
+    public String getUserid() {
```
Comment (dlazarenko-macrometa) [inline src/main/java/com/c8db/entity/UserQueryOptions.java:34]: typo: `getUserId`

File: src/main/java/com/c8db/entity/UserQueryOptions.java [hunk 2]
```diff
@@ -26,10 +26,20 @@
...
+    private String userid;
     private String name;
     private Map<String, Object> parameter;
     private String value;
 
+    public String getUserid() {
+        return userid;
+    }
+
+    public UserQueryOptions userid(final String userid) {
```
Comment (dlazarenko-macrometa) [inline src/main/java/com/c8db/entity/UserQueryOptions.java:38]: typo: `userId`

Comment (grainier-macrometa) [reply src/main/java/com/c8db/entity/UserQueryOptions.java:38]: @dlazarenko-macrometa It's intentional to make it compatible with the existing serialized `userid` field of `UserQueryEntity`.

File: src/main/java/com/c8db/entity/UserQueryOptions.java (hunk 1 above)
Comment (grainier-macrometa) [reply src/main/java/com/c8db/entity/UserQueryOptions.java:34]: Same as in other place

**Pull request merged**